scripts/security/whitesource_container_scan.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+source /root/cortx-re/solutions/kubernetes/functions.sh
+
+#Download WhiteSource Plugin 
+wget http://cortx-storage.colo.seagate.com/releases/cortx/security/whitesource/ws-k8s-agent.tar && tar -xvf ws-k8s-agent.tar >/dev/null 2>&1
+
+#Updating the configuration file
+sed -Ei 's,(url: ).*,\1"'"$WHITESOURCE_SERVER_URL"'",g' /root/ws-k8s-agent/helm-chart/values.yaml; sed -Ei 's,(apiKey: ).*,\1"'"$API_KEY"'",g' /root/ws-k8s-agent/helm-chart/values.yaml
```

Comment:
can you put `/root/ws-k8s-agent/helm-chart/values.yaml` this path in variable and use it wherever required?

Reply:
Updated /root/ws-k8s-agent/helm-chart/values.yaml with variable